ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1441,6 +1472,21 @@ RHDHC:
   implements:
   - CONTROL
 
+DH2XSDPC:
+  guid: "2e83d32e-360e-4ac2-afa4-5adb333ded0b"
+  description: "Supply air dewpoint temperature dehumidification control on dual cooling coil."
+  is_abstract: true
+  opt_uses:
+  - supply_air_relative_humidity_sensor
+  - failed_dehumidifier_alarm
```

Comment:
what would this alarm mean? you don't have a dehumidifier device, you have two coils

Reply:
good point, and we didn't have a BMS point for it, removed this alarm